### PR TITLE
Add a menu button always in the center of the screen

### LINKED ISSUE #10: Add menu view for selecting game settings
We need to have a menu accessible on screen to select number of players, starting life, resetting health, etc.
We should have this in place **BEFORE** we try to implement any of the features listed above.

### PR DESCRIPTION
**Pull request by gerritt-luoma**
My goal for this PR is to add a menu button to the center of the screen.  This button will open an overlay menu over the full screen that will eventually contain reset, player selection, color, and life count buttons.

Closes: #10

### CODE REVIEW

**Comment by gerritt-luoma** [comment]
# Done:
- Now have a button in the center of the screen at all times (should hopefully always stay there regardless of screen size)
- Menu now has state and is not visible when menu isn't toggles
- Main menu button toggles open menu when pressed.  This currently blows up the menu to be full screen with an opaque black background.  Can easily change!

# TODO
- [x] Add in dummy sub menu buttons for testing
- [x] Decide on styling for sub menu buttons (i.e. placing)
- [ ] Once we start adding in actual sub menu buttons we'll have to flesh out the menu state to handle all transitions

**Comment by gerritt-luoma** [comment]
Just added sub menu when button is pressed.  Now when the button is pressed the screen dims and sub menu buttons appear in a circle around the main button.  We should be able to easily make sub menus now based on the state.

File: App.tsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, View } from 'react-native';
 import { Provider } from 'react-redux';
 import { store } from './reducers/store';
-import Player from './components/Player/player';
+import Player from './components/Player/Player';
```
Comment (Insano12333) [inline App.tsx:5]: Looks like this will need to be lowercase, or the file changed to uppercase

**Review by Insano12333 — CHANGES_REQUESTED** [review]

File: App.tsx (hunk 1 above)
Comment (cole-daily) [inline App.tsx:5]: Looks like this will need to be lowercase, or the file uppercase

**Review by cole-daily — CHANGES_REQUESTED** [review]

File: App.tsx (hunk 1 above)
Comment (gerritt-luoma) [reply App.tsx:5]: Did you restart the server after pulling my changes?  It's working fine on my end

**Comment by gerritt-luoma** [comment]
Fixed issue with file casing.  Forgot you need to use `git mv`.

**Pull request merged**